recipe/meta.yaml

```diff
@@ -56,6 +56,7 @@ requirements:
     - tk 8.5.*  # [linux]
     - functools32  # [py2k]
     - setuptools
+    - libgcc  # [linux]
```

Comment:
That sounds right.

So we have been roughly following [manylinux]( https://www.python.org/dev/peps/pep-0513/#the-manylinux1-policy ) in terms of what we expect Linux to have. One of the things they expect there is `libstdc++`. So I think expecting `libstdc++` to be available on the VM or Docker image where `matplotlib` is installed is reasonable then. If we don't think this is reasonable, maybe we should revisit how we build things. Though this doesn't seem like a per feedstock discussion and seems more like an org level discussion.